src/components/SingleArticle.jsx

```diff
@@ -41,13 +42,30 @@ export default function SingleArticle(){
         if (!showComments) {
            
             getCommentsById(article_id).then((fetchedComments) => {
-                setComments(fetchedComments || []);
+                setComments(fetchedComments);
                 setShowComments(true);
             });
         } else {
             setShowComments(false);
         }
     }
+    function patchVote(event) {
+        event.preventDefault();
+        const newVotes = votes +1
+        setVotes(newVotes)
+        patchVotesByArticleId(article_id)
+            .then((updatedArticle) => {
+                setVotes(updatedArticle.votes)
+                setArticle((prevArticle) => ({
+                    ...prevArticle,
+                    votes: prevArticle.votes + 1, 
+                }));
+            })
+            .catch((err) => {
+                setError(err); 
```

Comment:
If I set the error from the votes handler, what do I want to happen? How will `err` inform the message I display to the user?